test/examples/make.jl

```diff
@@ -1,13 +1,15 @@
+using Compat
+
 # Defines the modules referred to in the example docs (under src/) and then builds them.
 # It can be called separately to build the examples/, or as part of the test suite.
 #
 # It defines a set of variables (`examples_*`) that can be used in the tests.
 # The `examples_root` should be used to check whether this file has already been included
 # or not and should be kept unique.
-isdefined(:examples_root) && error("examples_root is already defined\n$(@__FILE__) included multiple times?")
+@isdefined(examples_root) && error("examples_root is already defined\n$(@__FILE__) included multiple times?")
```

Comment:
`@isdefined` is not available on 0.6. We should have `isdefined(@__MODULE__, :examples_root)` instead. It should be possible to run the `make.jl` file separately as well.

Reply:
I put an `@isdefined` definition in `runtests.jl` that defaults to `isdefined` on 0.6 (otherwise this would have failed on 0.6).